classes/data-stores/ActionScheduler_DBStore.php

```diff
@@ -40,9 +40,10 @@ public function init() {
 	 * Save an action.
 	 *
 	 * @param ActionScheduler_Action $action Action object.
-	 * @param DateTime               $date Optional schedule date. Default null.
+	 * @param \DateTime              $date Optional schedule date. Default null.
```

Comment:
I realize there is a mix of styles in this file (in some places we fully qualify class names, in others we do not) but it seems unnecessary since we're not inside a namespace (and, the lack of `\` doesn't seem to be something that PHPCS was complaining about). 

In the interests of reducing line noise, what do you think of reverting these changes (that is, switching `\DateTime` back to `DateTime` and `\RuntimeException` to `RuntimeException`, etc)?

Reply:
Sure, it's done now.